lib/cucumber/formatter/pretty.rb

```diff
@@ -203,7 +203,8 @@ def table_cell_value(value, status)
         status ||= @status || :passed
         width = @table.col_width(@col_index)
         cell_text = escape_cell(value.to_s || '')
-        padded = cell_text + (' ' * (width - cell_text.unpack('U*').length))
+        text_width = defined?(Unicode) ? Unicode.width(cell_text, Cucumber.treats_ambiguous_as_fullwidth) : cell_text.unpack('U*').length
```

Comment:
This code looks quite similar to what's in `DataTable::Cells#width` below. If we move that logic into a `Cell#width` method, could we re-use that logic here by constructing an instance of a `Cell`?